cmd/get_logs.go

```diff
@@ -78,6 +83,8 @@ func newGetLogsCmd() *cobra.Command {
 	command.Flags().StringVar(&glc.linuxScriptPath, "linux-script", "", "path to the log collection script to execute on the cluster's Linux nodes (required)")
 	command.Flags().StringVarP(&glc.outputDirectory, "output-directory", "o", "", "collected logs destination directory, derived from --api-model if missing")
 	command.Flags().BoolVarP(&glc.controlPlaneOnly, "control-plane-only", "", false, "get logs from control plane VMs only")
+	command.Flags().StringVarP(&glc.blobServiceSASURL, "sas-url", "", "", "blob service SAS URL of the storage account on Azure or custom cloud to upload kubernetes logs")
```

Comment:
what if we go with `sa-sas-url` and `sa-container` (or `sa-container-name`), I am thinking that the `sa-` prefix will help user understand that these two go together

however, I still think one parameter will make things in long run, documentation should clarify that container name is expected and `validateArgs` should make sure the format is correct.